File: src/AuthDemo.ServiceModel/Types/Company.cs
using System;
using System.ComponentModel.DataAnnotations;
using ServiceStack.Common;
using ServiceStack.DataAnnotations;
using ServiceStack.DesignPatterns.Model;

namespace AuthDemo.ServiceModel.Types
{
	[Alias("COMPANY")]
	public partial class Company:IHasId<System.Int32>{

		public Company(){}

		[Alias("ID")]
		[Sequence("COMPANY_ID_GEN")]
		[PrimaryKey]
		[AutoIncrement]
		public System.Int32 Id { get; set;} 

		[Alias("NAME")]
		[StringLength(100)]
		public System.String Name { get; set;} 

		[Alias("TURNOVER")]
		public System.Single? Turnover { get; set;} 

		[Alias("STARTED")]
		public System.DateTime? Started { get; set;} 

		[Alias("EMPLOYEES")]
		public System.Int32? Employees { get; set;} 

		[Alias("CREATED_DATE")]
		public System.DateTime? CreatedDate { get; set;} 

		[Alias("GUID")]
		public System.Guid? Guid { get; set;} 


		
	}
}

File: src/AuthDemo.Host.Web/WebApp/modules/app/model/Company.js
Ext.define('AD.model.Company',{
	extend: 'Ext.data.Model',
	idProperty: 'Id',
    fields:[
    	{name: 'Id', type:'int'},
    	{name: 'Name', type:'string'},
    	{name: 'Turnover', type:'number'},
    	{name: 'Started' ,  type :'date', 
    			convert: function(v){ return  Aicl.Util.convertToDate(v)}},  				
    	{name: 'Employees' ,  type:'int'},
    	{name: 'CreatedDate' ,  type :'date', 
    	   		convert: function(v){ return  Aicl.Util.convertToDate(v)}},
    	{name: 'Guid' ,  type:'string'}
    ]
    
});
